Dockerfile

```diff
@@ -247,5 +249,11 @@ RUN --mount=type=cache,target=/root/.cache/pip \
 
 ENV VLLM_USAGE_SOURCE production-docker-image
 
+# port 8080 required by sagemaker, https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-inference-code.html#your-algorithms-inference-code-container-response
+ENTRYPOINT ["python3", "-m", "vllm.entrypoints.openai.api_server", "--port", "8080"]
+
+# the default image is `vllm-openai` which is identical to sagemaker without forcing the port
+from vllm-sagemaker as vllm-openai
+
 ENTRYPOINT ["python3", "-m", "vllm.entrypoints.openai.api_server"]
```

Comment:
Does this line not overwrite the entrypoint you declared earlier?

Reply:
Correct, so `docker build` has no change in behaviour from now. This gives the option of running `docker build --target vllm-sagemaker`, to create an image with the altered entrypoint.

Reply:
Thanks for the explanation!